### PR TITLE
Added pre-commit support and terraform-docs

### PR DESCRIPTION
**Pull request by onetwopunch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,8 +57,57 @@ resources of this module:
...
+| hostname | Host name of the bastion |
+| ip\_address | Internal IP address of the bastion host |
+| self\_link | Self link of the bastion host |
+| service\_account | The email for the service account created for the bastion host |
+
+<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+
 ## Contributing
 
+### Pre-Commit
```
Comment (morgante) [inline README.md]: While we *can* include pre-commit hooks I'd like us to follow the CFT standard which is to use the Docker commands to run these: https://github.com/terraform-google-modules/terraform-google-module-template/blob/master/terraform-google-%7B%7Bcookiecutter.module_name%7D%7D/CONTRIBUTING.md

Comment (morgante) [reply README.md]: In fact, it looks like this is causing linters to fail. Could we follow the standard tools please?

Comment (onetwopunch) [reply README.md]: Sure, but we use the same pre-commit scripts in the docker command. I just had the wrong options, which is why it was failing.

Comment (morgante) [reply README.md]: Got it. To be clear, I'm fine with us including pre-commit hooks but the contribution instructions should be based on the standard Docker workflow.

Comment (onetwopunch) [reply README.md]: Updated here: https://github.com/terraform-google-modules/terraform-google-bastion-host/pull/22

**Pull request closed without merging**